pilot/pkg/proxy/envoy/v1/authentication.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// AuthN filter configuration
+
+package v1
```

Comment:
Confused - either put it in deprecated, or don't add this. Please focus on v2, we're not trying to add new features to v1.

Reply:
Sure. I'm doing both. I'm doing this v1 version so that I can test (and clone implementation to v2) as I'm not sure v2 is in ready state.